### ISSUE TITLE
Reproduction on LSMDC DataSet

### ISSUE THREAD

**Issue by JingXiaolun** [ArrowLuo/CLIP4Clip#105]
Sorry to disturb you. When I reproduce the results on LSMDC dataset, I get worse results than those in paper. In the meanP experiment, the meanR is always around 200,  rather than around 60. The log is shown as follows:
![1686898632103](https://github.com/ArrowLuo/CLIP4Clip/assets/41146548/2eecb1e1-5a96-43a5-a2b4-b9a6e837646f)
The problem has confused me for a long time and I hope to get some advice for addressing this issue

**Comment by JingXiaolun** [comment]
@ArrowLuo

**Comment by sweet132** [comment]
Hi，have you solved the problem? When I reproduce the results on MSRVTT-9K dataset using seqTransf, I get 43.9% top 1, but if I compress videos first, it will be worse - 42.3% top-1.  I have no idea......

**Comment by shallowdream66** [comment]
I have observed your training logs and reproduced it. I found that the performance was poor only on the validation set, but returned to normal on the test set. I hope this will be helpful to you

**Comment by JingXiaolun** [comment]
@shallowdream66 ，the test set contains 1000 video clips,  and the validation  set contains 7408 video clips, do you mean using the 7408 video clips in validation process ?

**Comment by shallowdream66** [comment]
Yes，after each epoch it will be verified on the validation set, the checkpoint with the best results will be taken for testing, and the test results will return to normal.

**Comment by JingXiaolun** [comment]
@shallowdream66, thanks a lot, I will try it.

**Comment by JingXiaolun** [comment]
@shallowdream66 , can you share the training LSMDC log with me ?  I still can't reproduce the normal results in the test set
